website/azurerm.erb

```diff
@@ -387,6 +387,15 @@
               </ul>
             </li>
 
+            <li<%= sidebar_current("docs-azurerm-oms") %>>
+            <a href="#">OMS Resources</a>
+            <ul class="nav nav-visible">
+              <li<%= sidebar_current("docs-azurerm-log-analytics") %>>
```

Comment:
these strings are used for the highlight, so they need to be prefixed with the name above - as such could we make this: `docs-azurerm-oms-log-analytics`?

Reply:
Done!